frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -490,16 +502,15 @@ The interval value is in seconds and can range from 3 to 20 seconds.]]),
             },
         },
     }
-    for _, v in ipairs(long_press_action) do
+    for i, v in ipairs(self.long_press_action) do
         table.insert(menu_items.long_press.sub_item_table, {
-            text = v[1],
+            text = self.long_press_action_text[i],
             checked_func = function()
-                return G_reader_settings:readSetting("default_highlight_action", "ask") == v[2]
+                return v == G_reader_settings:readSetting("default_highlight_action", "ask")
```

Comment:
It indeed might feel a bit convoluted these `v[1]` and `v[2]`, but that's how we do mostly everywhere else.
You could just use `local name, text = v[1], v[2]` if you want it more readable.